src/components/TestResults/Screenshots.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { Modal, ModalClose, ModalContent } from 'mc-components'
+
+import Anchor from 'components/Anchor'
+import Screenshot from './Screenshot'
+
+const Screenshots = ({ screenshots }) => {
+  const [showScreenshots, setShowScreenshots] = useState(false)
+
+  return (
+    <Fragment>
+      <Anchor onClick={() => setShowScreenshots(true)}>Show Screenshots</Anchor>
+      <Modal onClose={() => setShowScreenshots(false)} show={showScreenshots}>
+        <ModalClose />
+        <div className="container">
+          <ModalContent>
+            <div className="mc-my-10">
+              <div className="row mc-mb-4">
+                {screenshots.current && (
+                  <div className={`col-6 ${screenshots.new ? '' : 'offset-3'}`}>
```

Comment:
@bernabe9 we have installed the `classnames` library on the project, why don't use it here?